enroll/certificates.go

```diff
@@ -1,22 +1,39 @@
 package enroll
 
 import (
+	"crypto/x509"
+	"encoding/pem"
 	"errors"
 	"golang.org/x/crypto/pkcs12"
 	"io/ioutil"
 )
 
-const PushTopicASN1 string = "0.9.2342.19200300.100.1.1"
+const (
+	certificatePEMBlockType string = "CERTIFICATE"
+	PushTopicASN1                  = "0.9.2342.19200300.100.1.1"
+)
 
-func GetPushTopicFromPKCS12(certPath string, certPass string) (string, error) {
+func GetPushTopicFromCert(certPath, certPass, keyPath string) (string, error) {
```

Comment:
Roger that.  The PKCS12/PushTopic code already existed and I minimally changed it. Noted nonetheless, though! I didn't catch that this was in the same package, so that can certainly change.